cryptlex/lexactivator/lexactivator.py

```diff
@@ -51,6 +51,7 @@ def __init__(self, user_license):
         self.allowed_deactivations = user_license.get("allowedDeactivations")
         self.key = user_license.get("key")
         self.type = user_license.get("type")
+        self.metadata = user_license.get("metadata",[])
```

Comment:
@muneebkq Please refer to https://github.com/cryptlex/lexactivator-python/blob/master/cryptlex/lexactivator/lexrelease.py#L30